easybuild/easyconfigs/l/LAMMPS/LAMMPS-23Jun2022-foss-2022a-kokkos.eb

```diff
@@ -37,7 +37,10 @@ dependencies = [
     ('gzip', '1.12'),
     ('cURL', '7.83.0'),
     ('HDF5', '1.12.2'),
-    ('tbb', '2021.5.0'),
+]
+if ARCH == 'x86_64':
+    dependencies += [('tbb', '2021.5.0'), ]
```

Comment:
can you move the tbb dep to the bottom of the list instead of splitting the list in 2?